### PR TITLE
Fix title when editing/creating surveys

### PR DESCRIPTION
**Pull request by ollfkaih**
Closes #https://github.com/webkom/lego/issues/2867

### CODE REVIEW

File: app/routes/surveys/components/SurveyEditor/SurveyEditor.js [hunk 1]
```diff
@@ -154,7 +154,7 @@ class SurveyEditor extends Component<Props, State> {
 
     return (
       <Content className={styles.detail}>
-        <Helmet title={titleField} />
+        <Helmet title={survey.id ? survey.title : 'Ny spørreundersøkelse'} />
```
[the remark this answers is not in the document]
Comment (ollfkaih) [reply app/routes/surveys/components/SurveyEditor/SurveyEditor.js:157]: Because it is a jsx node

[the remark this answers is not in the document]
Comment (eikhr) [reply app/routes/surveys/components/SurveyEditor/SurveyEditor.js:157]: Weren't you going to make this show the survey title as it was being typed?

[the remark this answers is not in the document]
Comment (ollfkaih) [reply app/routes/surveys/components/SurveyEditor/SurveyEditor.js:157]: Seemed difficult and might be easier to do when rewriting to react-final-form

[the remark this answers is not in the document]
Comment (eikhr) [reply app/routes/surveys/components/SurveyEditor/SurveyEditor.js:157]: True! I'll just assign you to that:)

**Review by eikhr — APPROVED** [review]

**Pull request merged**